cli/src/deploy/mod.rs

```diff
@@ -87,6 +88,7 @@ struct App {
 pub enum PostV1Error {
```

Comment:
Btw, I tried to update the `anycloud` dependency in the `alan` repo and it fails because of this enum, now. Can you add `#[derive(Debug)]` to it?